otx/algorithms/segmentation/adapters/mmseg/configurer.py

```diff
@@ -185,7 +185,6 @@ def configure_task(
         """Patch config to support training algorithm."""
         if "task_adapt" in cfg:
             logger.info(f"task config!!!!: training={training}")
-            cfg["task_adapt"].get("op", "REPLACE")
```

Comment:
```
self.task_adapt_op= cfg["task_adapt"].get("op", "REPLACE")
```
I'm not sure but doesn't it need to be changed as above?

Reply:
It was useless code line. It does not do any role and had no problem to remove this line.